### ISSUE TITLE
Single checkbox input is missing ARIA attributes

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#6303]
We want the input we tab to to have the `aria-*` attributes.

- The single checkbox is implemented by `checkbox-input.xbl`.
- It uses a `<xf:select appearance="full">`, which puts the `aria-*` attributes on a group around the checkboxes (instead, this issue calls for those attributes to be on the input itself, since we only have a single checkbox).
- Can this be implemented by having the `<xf:select>` itself support a single checkbox mode?

Related: https://github.com/orbeon/orbeon-forms/issues/6279.

[+1 from customer](https://3.basecamp.com/3600924/buckets/1956846/messages/7175826989)

**Comment by ebruchez** [comment]
We put `aria-*` attributes on a group because it makes sense when there are more than one item. Obviously, it doesn't make much sense when there is only one item.

We could detect the case where there is only one item statically, and produce different markup. Note that this is not a static itemset, because the label value can change. But we know that the number of items won't change dynamically.

**Comment by ebruchez** [comment]
If we detect a single-item full `xf:select`:

On `<span>` around items:

- don't output `role="group"`
- don't output `xforms-invalid`/`xforms-required`
- don't output `aria-describedby`

Instead, on `<input>`:

- output `xforms-invalid`/`xforms-required`
- output `aria-describedby`
